cli/medperf/utils.py

```diff
@@ -397,3 +398,39 @@ def filter_latest_associations(associations, entity_key):
 
     latest_associations = list(latest_associations.values())
     return latest_associations
+
+
+def check_for_updates() -> None:
+    """Check if the current branch is up-to-date with its remote counterpart using GitPython."""
+    repo = Repo(config.BASE_DIR)
+    if repo.bare:
+        logging.debug('Repo is bare')
+        return
+
+    logging.debug(f'Current git commit: {repo.head.commit.hexsha}')
+
+    try:
+        for remote in repo.remotes:
+            remote.fetch()
+
+        if repo.head.is_detached:
+            logging.debug('Repo is in detached state')
+            return
+
+        current_branch = repo.active_branch
+        tracking_branch = current_branch.tracking_branch()
+
+        if tracking_branch is None:
+            logging.debug("Current branch does not track a remote branch.")
+            return
+        if current_branch.commit.hexsha == tracking_branch.commit.hexsha:
+            logging.debug('No git branch updates.')
+            return
+
+        logging.debug(f'Git branch updates found: {current_branch.commit.hexsha} -> {tracking_branch.commit.hexsha}')
+        config.ui.print_warning('MedPerf client updates found. Please, reinstall client with '
+                                '`cd your/medperf/folder && git pull && pip install --upgrade -e ./cli`')
```

Comment:
```suggestion
        config.ui.print_warning('MedPerf client updates found. Please, update your MedPerf installation.')
```

Reply:
The reason is to have them search and follow our instructions of updating the installation, which involve additional steps